packages/contracts-bedrock/book/src/policies/release-process.md

```diff
@@ -0,0 +1,53 @@
+
+1. Checkout the commit
+1. Run `git tag <tag-string>`
+1. Run `git push origin <tag-string>`
+   Repo [rules](https://github.com/ethereum-optimism/optimism/rules/8196346?ref=refs%2Ftags%2Fop-contracts) require this is done by someone who is a [release-manager](https://github.com/orgs/ethereum-optimism/teams/release-managers). Once pushed a tag cannot be deleted, so please be sure it is correct.
+1. Create release notes in Github:
+   - Go to the [Releases page](https://github.com/ethereum-optimism/optimism/releases), enter or select `<tag-string>`
+     from the dropdown.
+1. Populate the release notes. If the tag is a release candidate, check the `Set as a pre-release`  option, and uncheck the
+   `Set as the latest release` option.
+1. Deploy the OPCM using the `op-deployer bootstrap implementations` [command](https://devdocs.optimism.io/op-deployer/user-guide/bootstrap.html),
+   this will write the addresses of the deployed contracts to `stdout` (or to disk if you provide an `--outfile` argument).
+   Do this on both Sepolia and Mainnet.
+1. In the superchain-registry edit the following files to add a new `[<tag-string>]` entry, with the addresses from the
+   previous step:
+   - [standard-versions-mainnet.toml](https://github.com/ethereum-optimism/superchain-registry/blob/main/validation/standard/standard-versions-mainnet.toml)
+   - [standard-versions-sepolia.toml](https://github.com/ethereum-optimism/superchain-registry/blob/main/validation/standard/standard-versions-sepolia.toml)
+1. Once the changes are merged into the superchain-registry, you can follow the [instructions](https://devdocs.optimism.io/op-deployer/reference-guide/releases.html#step-3-update-the-sr-with-the-new-release)
+   for creating a new release of `op-deployer`.
+
+## Implications for audits
+
+The process above should be followed to create an `-rc.1` release prior to audit. This will be the target commit for
+the audit. If any fixes are required by the audit results an Additional Release Candidate will be required.
+
+## Additional Release Candidates
+
+Sometimes fixes or additional changes need to be added to a release candidate version. In that case
+we want to ensure fixes are made on both the release and the trunk branch, without stopping development
+efforts on the trunk branch.
+
+The process is as follows:
+
+1. Make the fixes on `develop`. Increment the contracts semver as normal.
+1. Create a new release branch, named `proposal/op-contracts/vX.Y.Z` off of the rc tag (all subsequent `-rc` tags
+   will be made from this branch).
+1. Cherry pick the fixes from `develop` into the release branch, and increment the semver as normal. If this increment results in any contract's semver being equal to or greater than it is on `develop`, then the semver should immediately be increased on `develop` to be greater than on the release branch.
```

Comment:
This isn't super clear to me. If `develop` didn't progress _at all_ between one rc and the next, wouldn't we expect the patch, when applied to both `develop` and `proposal/op-contracts/vX.Y.Z` to result in identical semvers on the two branches? Why is that a problem? I don't really understand why the semver on `develop` needs to be strictly greater than it is on the `rc` (after all, this won't be the case when the first `rc` is cut).